coding/python_style_guide.rst

```diff
@@ -227,6 +227,20 @@ Style changes must be encapsulated in a distinct commit (see :ref:`git-commit-or
 
    :doc:`../docs/py_docs` provides guidelines for the :ref:`layout of docstrings <py-docstring-basics>`.
 
+.. _style-guide-license:
+
+Each Python file MUST contain the standard license preamble
+-----------------------------------------------------------
+
+A copyright and license block using :ref:`the standard text <pkg-doc-code-preamble>` MUST be included at the top of each file.
+This can be written as a Python comment.
+
+   .. code-block:: python
+
+      #
+      # <template text>
```

Comment:
Should something other than "<template text>" go in here? Is that supposed to be filled in automatically somehow?

Reply:
I was trying not to duplicate the text from the other primary definition. I'm not sure if I can get sphinx to insert the text from a standalone text file and prefix each line with a comment character.

Reply:
On the other hand, that boilerplate text shouldn't change without a big process (RFC, etc), so controlled duplication here is ok.

Reply:
It still seems fundamentally wrong to copy and paste it into three separate places.

Reply:
Okay, I'll look at how we can pull the content from here: https://github.com/lsst/templates/blob/master/file_templates/stack_license_py/example.py and https://github.com/lsst/templates/blob/master/file_templates/stack_license_cpp/example.cc